src/engraving/dom/edit.cpp

```diff
@@ -6642,7 +6650,7 @@ void Score::undoAddElement(EngravingItem* element, bool addToLinkedStaves, bool
             } else if (et == ElementType::PLAY_COUNT_TEXT) {
                 BarLine* bl = toBarLine(element->explicitParent());
                 Fraction tick = bl->tick();
-                Measure* m = score->tick2measureMM(tick - Fraction::eps());
+                Measure* m = score->tick2measure(tick - Fraction::eps());
```

Comment:
I'm wondering why this change? Cause usually the -MM version is the one we want here, cause if there are no mmRests it should just give you the normal measure (I think), and if there are then the barline actually belongs to the -MM measure so that's what you need.